.readthedocs.yaml

```
@@ -1,14 +1,16 @@
-  environment: environment.yml
+    python: "3.11"
+  jobs:
+    create_environment:
+      - asdf plugin add pixi
+      - asdf install pixi latest
+      - asdf global pixi latest
+    install:
+      - pixi install
+    build:
+      html:
+        - pixi run sphinx-build -T -b html docs $READTHEDOCS_OUTPUT/html
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)